src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/UrlParamBasedImgTagAttrInjection.java

```diff
@@ -204,7 +206,8 @@ public UrlParamBasedImgTagAttrInjection() {
             description =
                     "XSS_QUOTES_AND_WITH_HTML_ESCAPE_PLUS_FILTERING_ON_INPUT_SRC_ATTRIBUTE_IMG_TAG")
     @VulnerableAppRequestMapping(
-            value = LevelConstants.SECURE,
+            value = SecureConstants.SECURE_1,
```

Comment:
make value = LevelConstants.LEVEL_*

Reply:
Doing that results in a run-time exception thrown by having the same value in the controller scope. This behaviour prevents ambiguity. Introducing duplicates would mean that identical endpoint would have to be handled via different methods. Additonally, this is connected to https://github.com/SasanLabs/VulnerableApp/pull/278#discussion_r623483099

Reply:
no no wait, * was the next level number i.e. `Level_7`. please try this and it will work.

Reply:
It will, but wasn't this endpoint supposed to be marked as _secure_?

Reply:
As recently discussed, a single level will be unique in the controller scope, so I have changed it per your suggesion. Resolved in 3211f26